Package.swift

```diff
@@ -43,6 +43,9 @@ let package = Package(
                 "Monitors",
                 "Tools"
             ],
+            resources: [
+                .copy("PrivacyInfo.xcprivacy")
```

Comment:
For now, I added a kind of "monilit" privacy report in this module, since SPM modules need work. It shows the data collected in all KSCrash submodules.